yarn.lock

```diff
@@ -0,0 +1,4207 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
It is a yarn generated file that helps speed up installs and, consequently, CI runs.